CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - Fixed gas assertion in `*_transfer_call` implementations of FT and NFT standards to only require what's needed. [PR 760](https://github.com/near/near-sdk-rs/pull/760)
 - Fixed events being emitted in FT standard to include refund transfers and burn events. [PR 752](https://github.com/near/near-sdk-rs/pull/752)
 
+### Fixed
+- Remove `Clone` implementation for `Promise` (error prone) https://github.com/near/near-sdk-rs/pull/783
```

Comment:
Probably better in the `Removed` section?

Reply:
Yeah, I figured since it's an actual bug I would include here, guess it could go either way, though